### PR TITLE
Faq

### PR DESCRIPTION
**Pull request by jospoortvliet**
@tomneedham @dragotin @danimo what do you think about the delta-sync part... if OK, merge.

### CODE REVIEW

File: page-faq.php [hunk 1]
```diff
@@ -76,6 +77,14 @@
...
 <p>Note that <a href="http://owncloud.org/blog/introducing-bounty-source-for-owncloud/" target="_blank">Bountysource</a> offers a way of backing feature requests with your wallet.</p>
 
+<a name="upgrade"></a>
+<h3>I want to upgrade my ownCloud installation to the new release, how do I do that?</h3>
+<ul>
+	<li>The manual upgrade process is described in the <a href="<?php echo $DOCUMENTATION_ADMIN; ?>/maintenance/upgrade.html" target="_blank">ownCloud Documentation</a>.</li>
+	<li>If you installed ownCloud using Linux distribution packages and added the repository, an update will get them. You can also manually grab the new versions from <a href="/install/" target="_blank">owncloud.org/download</a></li>
+	<li>The build in Update app will usually have new ownCloud releases available for upgrade some time in the week after a major release.</li>
```
Comment (tomneedham) [inline page-faq.php]: build -> built

**Comment by jospoortvliet** [comment]
@tomneedham should be good to go now ;-)

**Comment by tomneedham** [comment]
Do we want any word from @dragotin ?

**Comment by jospoortvliet** [comment]
that'd be nice but he doesn't like me much so he tends to ignore me ;-)

**Comment by tomneedham** [comment]
Cool :) Let's go!

**Comment by dragotin** [comment]
@jospoortvliet  Dont say these things.

Apart from that nice, except that I think the bounty-source thing is ... well. You know it ;-P

**Comment by jospoortvliet** [comment]
@dragotin yeah yeah, I know... You grumpy man... ;-)

**Pull request merged**